content/guides/references/experiments.md

```diff
@@ -39,11 +39,12 @@ creating `e2e` and `component` objects inside your Cypress configuration.
+| -------------------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `experimentalStudio`                   | `false` | Generate and save commands directly to your test suite by interacting with your app as an end user would.                                                                                                                                                                                                                                                                                                                                                              |
+| `experimentalRunAllSpecs`              | `false` | Enables the "Run All Specs" UI feature, allowing the execution of multiple specs sequentially.                                                                                                                                                                                                                                                                                                                                                                         |
+| `experimentalOriginDependencies`       | `false` | Enables support for `require`/`import` within `cy.origin`.                                                                                                                                                                                                                                                                                                                                                                                                             |
+| `experimentalUseDefaultDocumentDomain` | `null`  | Disables injecting `document.domain` into `text/html` pages. The `document.domain` configuration has been known to cause issues with certain sites, including Google and Salesforce. Enabling this option no longer allows for default sub domain navigations, and will require the use of `cy.origin()`. This option takes an array of strings/minimatch globs. Our recommended configuration is `['*.salesforce.com', '*.force.com', '*.google.com', 'google.com']`. |
```

Comment:
I would have expected more documentation for this experiment. If we know why google doesn't work and give our high-level understanding of salesforce it'll help people understand if they should enable this. I think we probably only want users to turn this on if absolutely necessary right?

Also an example of what changes might be needed where it worked without origin then needed origin might help users.

Reply:
> I think we probably only want users to turn this on if absolutely necessary right?

I would say so, or we leverage as a diagnostic tool. 

Do we have a place where we can further document the experiment to add some more verbosity? Right now the column already looks bloated and is a bit hard to read. I'd almost like to trim this down, and have a link to more details about the experiment that users can discover. Otherwise, we'd have to mostly guide through GitHub which I am not a big fan of.

Reply:
@emilyrohrbough I took a stab at this in 09e859c92f5bf80e8e6de39fa7e4544a8f57b101 and removed the reference in the web security page. I figured since this is a pretty small sample size, mentioning in the experiments and cross origin testing guide at the end should suffice